srcpkgs/libsecp256k1/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'libsecp256k1'
+pkgname=libsecp256k1
+version=20190311+ee99f12
+revision=1
+_commit=ee99f12f3d8af18bbb9b74cd31907ac1979742ec
```

Comment:
Please ask upstream for a tagged release, I doubt this will be merged otherwise

Reply:
https://github.com/bitcoin-core/secp256k1/issues/286